packages/search-in-workspace/src/common/search-in-workspace-interface.ts

```diff
@@ -53,6 +53,11 @@ export interface SearchInWorkspaceOptions {
      * Glob pattern for matching files and directories to exclude the search.
      */
     exclude?: string[];
+    /**
+     * Whether symlinks should be followed while searching.
+     * See the vscode setting `"search.followSymlinks"`.
```

Comment:
We probably do not need to reference the vscode setting here, I think it's self-explanatory.